source/dotnet/Library/AdaptiveCards/ParseContext.cs

```diff
@@ -1,7 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
 using System;
 using System.Collections.Generic;
 using System.Linq;
-using Newtonsoft.Json;
+using System.Threading;
```

Comment:
We can problably remove this

Reply:
thanks

---
In reply to: [340381126](https://github.com/microsoft/AdaptiveCards/pull/3546#discussion_r340381126) [](ancestors = 340381126)